### PR TITLE
Fixed version plus a background fix

### PR DESCRIPTION
**Pull request by Fluffy-hyena**
- the background works with any theme now

### CODE REVIEW

File: Plugins/dateViewer/dateViewer.plugin.js [hunk 1]
```diff
@@ -1,7 +1,204 @@
-//META{"name":"DateViewer","displayName":"Date Viewer","website":"https://github.com/hammy1/BDStuff/tree/master/Plugins/dateViewer","source":"https://raw.githubusercontent.com/hammy1/BDStuff/master/Plugins/dateViewer/dateViewer.plugin.js"}*//
+/**
+ * @name DateViewer
+ * @author ezeholz
```
Comment (ezeholz) [inline Plugins/dateViewer/dateViewer.plugin.js:3]: This seems to be my fork. That version is a complete modification of the original program. You could create the same PR in that fork, or made those changes in the original part of the code.

**Review by ezeholz — CHANGES_REQUESTED** [review]

File: Plugins/dateViewer/dateViewer.plugin.js (hunk 1 above)
Comment (Fluffy-hyena) [reply Plugins/dateViewer/dateViewer.plugin.js:3]: oh, sorry I didnt know. Im closing this pr anyways I was in the process of adding things to make it work 100% of the time but the hsla conversions messed me up too much

Comment (Fluffy-hyena) [reply Plugins/dateViewer/dateViewer.plugin.js:3]: well idk I mightve found a work around. If I do Ill do a pr on yours rather than the main

Comment (ezeholz) [reply Plugins/dateViewer/dateViewer.plugin.js:3]: Go ahead! I already saw your change, and seems like a good addition. Waiting for it :D

**Pull request closed without merging**